frameworks/PHP/symfony/app/Resources/views/base.html.twig

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
```

Comment:
if we really want to reduce the size of the output, this is valid HMTL5:
```
<!DOCTYPE html>
<title>{% block title %}Welcome!{% endblock %}</title>
{% block body %}{% endblock %}
```

Reply:
Yes but it's not conform to the requirements: 
https://www.techempower.com/benchmarks/#section=code&hw=ph&test=fortune